python/pyarrow/src/arrow/python/numpy_to_arrow.cc

```diff
@@ -223,10 +223,16 @@ class NumPyConverter {
   Status Visit(const NullType& type) { return TypeNotImplemented(type.ToString()); }
 
   // NumPy ascii string arrays
+  template <typename T>
+  Status VisitBinary(T* builder);
   Status Visit(const BinaryType& type);
+  Status Visit(const LargeBinaryType& type);
 
   // NumPy unicode arrays
+  template <typename T>
+  Status VisitString(T* builder);
```

Comment:
Small nitpick, but could you move those declarations of the helpers into `protected:` (eg just below `VisitNative` definition) 

(I don't think anyone outside of pyarrow is using this, but just to keep it consistent)

Reply:
Just the `VisitBinary` and `VisitString` declarations specifically?

Reply:
Yes, exactly as what you did